app/javascript/dashboard/components/widgets/conversation/linear/CreateOrLinkIssue.vue

```diff
@@ -57,6 +63,20 @@ const { t } = useI18n();
 
 const selectedTabIndex = ref(0);
 
+const timeAgo = time => {
+  const unixTime = fromUnixTime(time);
+  return formatDistanceToNow(unixTime, { addSuffix: true });
+};
+const title = computed(() => {
```

Comment:
Won't this keep changing? Suppose I linked a month old conversation, and then take a look at it 3 months later. The title will still show "A month ago"

Reply:
Good catch. We can update the preview link, but it requires some effort. For now, let's display only the conversation ID and contact name.

Reply:
Yes! Rest LGTM, let's get this PR merged 🔥